src/utils/log.h

```diff
@@ -30,21 +30,67 @@
 #ifndef G_UTILS_LOG_H
 #define G_UTILS_LOG_H
 
+#include <cstdio>
+#include <string>
+#include <type_traits>
+#include <utility>
 
-namespace giada {
-namespace u {
-namespace log 
-{
-/* init
-Initializes logger. Mode defines where to write the output: LOG_MODE_STDOUT,
-LOG_MODE_FILE and LOG_MODE_MUTE. */
+#include "utils/fs.h"
+#include "core/const.h"
 
-int init(int mode);
+namespace giada::u {
 
-void close();
+struct log {
+  static inline FILE* f;
```

Comment:
We use tabs for indentation (#335 will fix this stuff when implemented)

Reply:
Why not.
So everybody has the exact same definition of tab behavior.
I pushed a commit with tabs instead.